test/PluginTest.php

```diff
@@ -139,9 +143,9 @@ private function getMockComposer()
             'accessor-generator' => [
                 'database.table.column' => [
                     'public-key' => 'public',
-                    'private-key' => 'secret'
-                ]
-            ]
+                    'private-key' => 'secret',
+                ],
+            ],
```

Comment:
... While you add them here? Please be consistent.

Reply:
This is a multi-line array, which should have trailing comma's. This is consistent with our codestyle.